src/shared/components/home/tagline-form.tsx

```diff
@@ -0,0 +1,190 @@
+import { Component, InfernoMouseEvent, linkEvent } from "inferno";
+import { EditSite, GetSiteResponse } from "lemmy-js-client";
+import { i18n } from "../../i18next";
+import { WebSocketService } from "../../services";
+import { capitalizeFirstLetter, myAuth, wsClient } from "../../utils";
+import { HtmlTags } from "../common/html-tags";
+import { Icon, Spinner } from "../common/icon";
+import { MarkdownTextArea } from "../common/markdown-textarea";
+
+interface TaglineFormProps {
+  siteRes: GetSiteResponse;
+}
+
+interface TaglineFormState {
+  siteRes: GetSiteResponse;
+  siteForm: EditSite;
+  loading: boolean;
+  editingRow: number | null;
+}
+
+export class TaglineForm extends Component<TaglineFormProps, TaglineFormState> {
+  state: TaglineFormState = {
+    loading: false,
+    siteRes: this.props.siteRes,
+    editingRow: null,
+    siteForm: {
+      taglines: this.props.siteRes.taglines?.map(x => x.content),
+      auth: "TODO",
+    },
+  };
+  constructor(props: any, context: any) {
+    super(props, context);
+  }
+  get documentTitle(): string {
+    return i18n.t("taglines");
+  }
+
+  componentWillReceiveProps() {
+    this.setState({ loading: false });
+  }
+
+  render() {
+    return (
+      <div className="col-12">
+        <HtmlTags
+          title={this.documentTitle}
+          path={this.context.router.route.match.url}
+        />
+        <h5 className="col-12">{i18n.t("taglines")}</h5>
+        <div className="table-responsive col-12">
+          <table id="taglines_table" className="table table-sm table-hover">
+            <thead className="pointer">
+              <th></th>
+              <th style="width:121px"></th>
```

Comment:
Don't hard code widths like this. Use responsive bootstrap tables.

Reply:
This is pulled from emoji table, used for same reason, just to give a min width to the column to have two buttons on the same row.

Reply:
Okay, but I highly suggest you read over how bootstrap does responsive forms: https://getbootstrap.com/docs/5.0/forms/overview